test/util/backend_utils.hpp

```diff
@@ -0,0 +1,205 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#ifdef NGRAPH_UNIT_TEST_OPENVINO_ENABLE
+#include <ie_core.hpp>
+#include <string>
+#include "ngraph/ngraph.hpp"
+
+using namespace std;
+using namespace ngraph;
+using namespace InferenceEngine;
+
+Blob::Ptr fill_blob(SizeVector shape, std::vector<float> data);
+
+class Handle;
+
+namespace runtime2
```

Comment:
do you want to make this more descriptive? "ov-runtime" or "ieplugin-runtime"?

Reply:
Done, thanks